code/game/gamemodes/antag_spawner.dm

```diff
@@ -127,6 +141,7 @@
 	var/TC_cost = 0
 	var/borg_to_spawn
 	var/list/possible_types = list("Assault", "Medical")
+	var/spawn_type = BORG_TELE

```

Comment:
I didnt want to change the borg tele path because I am unsure what that would do to stat tracking.